cmd/vdev_id/vdev_id

```diff
@@ -276,6 +276,26 @@ sas_handler() {
 		d=$(eval echo \${$i})
 		SLOT=`echo $d | sed -e 's/^.*://'`
 		;;
+	"SES")
```

Comment:
[nit] for consistency with "lun" let's go with a lower case "ses".

Reply:
Initially I did write `ses` in lower case, but I decided it was too easily confused with `sas`, and that `SES` was much clearer.  That is, clarity trumped consistency.  Do you agree?

(I'll address the other points on Monday.)

Reply:
I can see that, but in this case I don't think it will result in too much confusion since `sas` is not a valid value for `slot`.

Reply:
OK, now lowercased.